app/javascript/src/hyacinth_v1/components/batch_imports/BatchImportShow.js

```diff
@@ -47,7 +47,10 @@ function BatchImportShow() {
     batchImport: {
       numberOfInProgressImports: inProgress,
       numberOfPendingImports: pending,
-      numberOfFailureImports: failure,
+      numberOfCreationFailureImports: creationFailure,
+      numberOfUpdateFailureImports: updateFailure,
+      numberOfPersistFailureImports: persistFailure,
+      numberOfPublishFailureImports: publishFailure,
```

Comment:
Do you mind making these variable names plural?  e.g. creationFailure -> creationFailures

Reply:
I don't! I was just trying to follow the pattern of the predecessor variable.

Reply:
Ben and I talked this over and we'll go with `Count` instead:

inProgressCount, pendingCount, creationFailureCount, etc.